commit 944d1f351778cb9f99ccab50ba8e80067e3c85f1
Author: q66 <q66@chimera-linux.org>
Date:   Tue Oct 17 16:38:55 2023 +0200

    fix some syscalls being disallowed in the seccomp sandbox
    
    Ref https://gitlab.gnome.org/GNOME/tracker-miners/-/issues/285#note_1871676

diff --git a/src/libtracker-miners-common/tracker-seccomp.c b/src/libtracker-miners-common/tracker-seccomp.c
index c46b992..80bb2fb 100644
--- a/src/libtracker-miners-common/tracker-seccomp.c
+++ b/src/libtracker-miners-common/tracker-seccomp.c
@@ -47,14 +47,14 @@
 #endif
 
 #define ALLOW_RULE(call) G_STMT_START { \
-	int allow_rule_syscall_number = seccomp_syscall_resolve_name (G_STRINGIFY (call)); \
+	int allow_rule_syscall_number = seccomp_syscall_resolve_name (#call); \
 	if (allow_rule_syscall_number == __NR_SCMP_ERROR || \
 	    seccomp_rule_add (ctx, SCMP_ACT_ALLOW, allow_rule_syscall_number, 0) < 0) \
 		goto out; \
 } G_STMT_END
 
 #define ERROR_RULE(call, error) G_STMT_START { \
-	int error_rule_syscall_number = seccomp_syscall_resolve_name (G_STRINGIFY (call)); \
+	int error_rule_syscall_number = seccomp_syscall_resolve_name (#call); \
 	if (error_rule_syscall_number == __NR_SCMP_ERROR || \
 	    seccomp_rule_add (ctx, SCMP_ACT_ERRNO (error), error_rule_syscall_number, 0) < 0) \
 		goto out; \
